style.css

```diff
@@ -0,0 +1,167 @@
+html {
+  background-color: #ffe6e6;
+  color: black;
+}
+
+nav {
+  text-align: center;
+}
+
+table {
+  border: 2px solid black;
+}
+
+form {
+  width: 50%;
+  border: 2px solid black;
+}
+
+img {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+#navigation {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  overflow: hidden;
+  background-color: #333;
+}
+
+#navigation li {
+  float: left;
+}
+
+#navigation li a {
+  display: block;
+  color: white;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+}
+
+#navigation li a:hover {
+  background-color: #111;
+}
+
+#PatSalmonCookies {
+  text-align: center;
+  font-family: 'Playfair Display', serif;
+  font-size: 30px;
+  padding-bottom: 30px;
+  font-weight: bold;
+}
+
+#header3color {
+  color: #A1BAA5;
+}
+
+.mainPageBlock {
+  text-align: center;
+
+}
+
+.h_line {
+  width: 100%;
+  height: 1px;
+  background: #fff;
+}
+
+.pcopy {
+  text-align: left;
+  font-family: 'Playfair Display', serif;
+  font-size: 18px;
+  padding-bottom: 10px;
+}
+
+#MainDiv {
+  text-align: left;
+  font-family: 'Playfair Display', serif;
+  font-size: 18px;
+  padding-bottom: 40px;
+  padding-left: 200px;
+}
+
+#headers {
+  padding-bottom: 100px;
+  text-align: left;
+  font-family: 'Playfair Display', serif;
+}
+
+#Header1 {
+  font-family: 'Playfair Display', serif;
+  font-size: 40px;
+}
+
+#Header2 {
+  font-family: 'Playfair Display', serif;
+  font-size: 30px;
+}
+
+#Header3 {
+  font-family: 'Playfair Display', serif;
+  font-size: 20px;
+  color: #A1BAA5;
+}
+
+#bodyCopyDiv {
```

Comment:
Couple of things, other than the CSS is looking pretty solid!

1. You have quite a few redundant rules that are applied in multiple places using different selectors. Flip that script and DRY out your CSS. Put those rules into one rule set using a class, and then apply the class to those elements which need it's rules. 

2. Mentioned briefly above, start to think about when and where you really need to use an ID as a selector for CSS. I would rather you focus on the reusable rules and then use classes as selectors so you can reuse and stay DRY.